backend/.env

```
@@ -1,5 +1,5 @@
-DB_PASSWORD=D1ymf8wyQEGthFR1E9xhCq
-DB_USER=LTIdbUser
-DB_NAME=LTIdb
+DB_PASSWORD=514071
+DB_USER=marcelo
+DB_NAME=marcelo_db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PASSWORD key

(UnorderedKey)